<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedBack</class>
 <widget class="QWidget" name="FeedBack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FeedBack</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>726</width>
        <height>1500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>1500</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>反馈信息</string>
         </property>
         <widget class="QLabel" name="label_41">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请写下你信息要修改的地方：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_commit">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>1350</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_feedback">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>681</width>
            <height>331</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_42">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>450</y>
            <width>361</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加图片（可选，如要上传，至少上传3张）：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>510</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_up1">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>660</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>华文彩云</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>790</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_up2">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>940</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>华文彩云</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>1060</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_up3">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>1210</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>华文彩云</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
